programs/hh-escrow/src/state/market.rs

```diff
@@ -49,16 +47,22 @@ pub struct Market {
     pub no_account_bump: u8,
     /// Whether the resolver has acknowledged this market.
     pub acknowledged: bool,
-    /// The URI to the market info (i.e. title, description)
+    /// The URI to the market's info (i.e. title, description)
     pub uri: UriResource,
 }
 
 impl Market {
-    #[allow(clippy::identity_op)]
-    pub const LEN: usize =
-        (5 * PUBKEY_BYTES) + (7 * 8) + 4 + Outcome::LEN + 1 + (2 * 1) + 1 + UriResource::LEN;
+    pub const LEN: usize = 5 * 32 + 7 * 8 + 4 + 1 + 1 + 2 * 1 + 1 + UriResource::LEN;
```

Comment:
We don't want/need to use constants where we can?

Reply:
We definitely do, but this PR just reverts a bunch of code to how it was when it was deployed to mainnet.